src/pages/recipes.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Link } from "gatsby";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+
+const Wrapper = styled.div`
+  margin: 50px 20px;
+  padding: 10px;
+`;
+const Image= styled.img`
+width: 600px;
+height: 500px;
+`
+const Recipes = ({ data }) => {
+  const page = data.allContentfulRecipes.edges;
+  return (
+    <Layout>
+      <SEO title="Recipes" />
+      {page.map(({ node }) => {
+        const title = node.title || node.slug;
```

Comment:
no need to create the variable, you can render this condition directly as the child of the Link component